From c9d4152db8fb929e62a86a75474dcca6f2425022 Mon Sep 17 00:00:00 2001
From: Robby Workman <rworkman@slackware.com>
Date: Tue, 27 May 2014 22:27:00 -0500
Subject: [PATCH 1/4] brewtarget.desktop.in: Define only the icon name (no
 path/extension)

This commit defines the icon name as only "brewtarget" - it will
require subsequent commit(s) to change the icon name and location
to which it installs.
---
 brewtarget.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/brewtarget.desktop.in b/brewtarget.desktop.in
index c3ae613..cc96055 100644
--- a/brewtarget.desktop.in
+++ b/brewtarget.desktop.in
@@ -4,7 +4,7 @@ Exec=brewtarget
 Name=Brewtarget
 GenericName=Beer calculator
 X-KDE-StartupNotify=true
-Icon=${brewtarget_ICONS_LOCATION}/brewtarget_icon.svg
+Icon=brewtarget
 Terminal=false
 Type=Application
 
-- 
1.8.4

